token-swap/js/src/util/send-and-confirm-transaction.ts

```diff
@@ -10,10 +13,11 @@ export function sendAndConfirmTransaction(
   title: string,
   connection: Connection,
   transaction: Transaction,
+  options: ConfirmOptions,
   ...signers: Array<Account>
 ): Promise<TransactionSignature> {
   return realSendAndConfirmTransaction(connection, transaction, signers, {
```

Comment:
`realSendAndConfirmTransaction` already takes in `ConfirmOptions` as the final parameter, which means this function is pretty much exactly the same.  I'm thinking we can remove this version of `sendAndConfirmTransaction` and always use the official version from `@solana/web3.js`.  What do you think?

Reply:
Yeah sounds good, added this.